content/pandas/concepts/dataframe/terms/reset-index/reset-index.md

```diff
@@ -0,0 +1,167 @@
+---
+Title: '.reset_index()'
+Description: 'Resets the index of a DataFrame to be continuous' 
+Subjects: 
+  - 'Data Science'
+  - 'Pandas'
+Tags:
+  - 'Methods'
+  - 'Pandas'
+  - 'Data Science'
+CatalogContent:
+  - 'learn-python'
+  - 'paths/data-science'
+---
+
+Sometimes there is a need to modify a DataFrame object to better meet a certain use case. When these modifications are applied they often leave smaller DataFrames with indexes that are messy and not continuous. Resetting the index with **`.reset_index()`** fixes these issues.
+
+## Syntax
+
+```py
+import pandas as pd
+df.read_csv('some_csv.csv', index_col = 'Account Number')
+df.reset_index()
+```
+
+In this syntax example the `index_col = Account Number` parameter sets the index of the DataFrame to a column named Account Number and then `df.reset_index()` resets the DataFrame index. 
+
+### `.reset_index()` Parameters
+
+The `.reset_index()` method provides many parameters which allow for more precision when resetting a DataFrame index. The following parameters will be used in the examples later in this article.
+
+- *drop:* The drop parameter can be set to True or False and is set to False by default. When this parameter is set to True it replaces the previous DataFrame index with the new index provided by `.reset_index()` otherwise it sets the new index in front of the old index.
+- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes `.reset_index()` makes to the current instance of the DataFrame otherwise it creates a new DataFrame instance with the changes applied to that DataFrame.
+
+## Actions Which Cause Indexing Issues 
+
+Common examples include but are not limited to:
+
+- Changing the order of columns.
+- Using a column or multiple columns of the database as an index instead of the default index pandas provides and then later needing a numbered index.
+- Filtering out certain rows of the DataFrame based on the value of a column of data.
+- Changing the order of rows of data by sorting those rows by the values of a certain column of data such as Date or Employee ID Number.
+- Adding columns and or rows of data to the DataFrame that were not present in the original DataFrame.
+
+## Examples
+
+In the following examples imagine working on a project about dogs and wanting to use data from an animal shelter about dogs in that shelter but the data in the .csv file has data about cats and dogs.
+To follow along a copy of the [Austin_Animal_Center_intakes.csv](https://www.kaggle.com/datasets/jackdaoud/animal-shelter-analytics) can be downloaded from Kaggle.
```

Comment:
```suggestion
To follow along a copy of the [Austin_Animal_Center_intakes.csv](https://data.austintexas.gov/Health-and-Community-Services/Austin-Animal-Center-Intakes/wter-evkm) can be downloaded from city of Austin data portal.
```

Reply:
This is a small change, but ideally we link back to a source that has an explicit (public) license. In this case Kaggle is an intermediary and the source of the data presented on this page could change.